resources/views/platform/components/beaten-games-leaderboard/meta-panel/index.blade.php

```diff
@@ -5,6 +5,16 @@
     'selectedConsoleId' => null,
 ])
 
+<?php
+// Arduboy, WASM-4, and Uzebox are homebrew consoles.
+// They do not have games that would conventionally be considered "retail".
+$allowsRetail = (
+    $selectedConsoleId !== 71
+    && $selectedConsoleId !== 72
+    && $selectedConsoleId !== 80
+);
```

Comment:
suggest putting this logic in the `System` model (see also `isGameSystem`).

Reply:
Good idea. Addressed in latest with two new static methods in `System`: `getHomebrewSystems()` and `isHomebrewSystem()`. This lets us rely on the `System` model for both the front-end and the back-end logic.